ci-operator/step-registry/workers-rhel/pre-hook/gcp/workers-rhel-pre-hook-gcp-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
+
+cat > scaleup-pre-hook-gcp.yaml <<- 'EOF'
+- name: Ensure RHEL8 machines on GCP have 'checkmodule'.
+  hosts: new_workers
+  any_errors_fatal: true
+  gather_facts: false
+  tasks:
+  - name: install checkpolicy - rhel8
+    yum: name=checkpolicy state=present
```

Comment:
I think we can merge the step to ci-operator/step-registry/workers-rhel/pre-hook/sshtunnel/workers-rhel-pre-hook-sshtunnel-commands.sh